Added ScubaGoggles Not Found Error to ReadME (#180)
cisagov/ScubaGoggles: README.md

```
Ensure that you consented to the following API scopes as a user with the proper [permissions to consent](#permissions) and have enabled the required [APIs and Services](#create-an-oauth-credential).

### Scubagoggles Not Found 
If an error similar to the one below appears: 
```
command not found: scubagoggles
```

Ensure that you have properly [configured the virtual environment](#installing-in-a-virtual-environment) and have activated the virtual environment using the OS appropriate commands. 

> To run scubagoggles without using a virutal environment, you can replace the `scubagoggles` command with `python scuba.py` . 

Reviewer: You can still run `scubagoggles gws ...` without a virtual environment though. You just do all of your `pip installs` outside of the virtual environment and it should work the same as it would inside the virtual environment.

The following might be more precise:
```suggestion
Alternatively, to run scubagoggles without installing it as a package, you can replace the `scubagoggles` command with `python scuba.py`. 
```
